tests/test_opSimOrdering.py

```diff
@@ -0,0 +1,76 @@
+import os
```

Comment:
True. I can speed it up by using a `setUpClass`, but I will have trouble doing it this week since I am away. Another speedup is using a small OpSim output which I use for my testing in other repos. The small OpSim output I use is [here](https://github.com/rbiswas4/OpSimSummary/blob/master/opsimsummary/example_data/enigma_1189_micro.db) and is created by the [script](https://github.com/rbiswas4/OpSimSummary/blob/master/scripts/make_smallOpSim.py)